src/RestClient/endpoints/V3/Themes/Themes.ts

```diff
@@ -0,0 +1,58 @@
+import { AxiosInstance } from 'axios';
+
+import { AxiosPromise } from '../../../../types';
+
+import type { Theme } from './types';
+
+export const themesPath = 'v3/themes';
```

Comment:
🍹 Is this path working for you in testing?

In the parent RestClient.ts class, we do not provide a trailing forward slash at the end of `baseUrl`: https://github.com/bc-nekisham/bigcommerce-api-node/blob/5f6be77c59cc21dce84adbc4e556d29ebbd69349/src/RestClient/RestClient.ts#L22

I would expect the themesPath to begin with a forward slash. With that said, this could be something Axios adds natively. You might have to do some digging to find out if Axios appends forward slashes to the end of `baseUrl`'s by default.

Reply:
Path was working, but updated it with / slash for consistency